src/app/ubs-admin/components/ubs-admin-orders/ubs-admin-orders.component.html

```diff
@@ -0,0 +1,112 @@
+<div class="wrapper">
+  <div class="mainHeader">Ваші замовлення</div>
+  <div *ngIf="isAnyOrders" class="ifEmpty"><span>Ви ще не зробили замовлень у нас.</span><a href="#">Зробити замовлення</a></div>
+  <div *ngIf="!isAnyOrders" class="cards">
+    <div class="headerList">
+      <div>№ замовлення</div>
+      <div>Статус</div>
+      <div>Сума оплати</div>
+    </div>
+    <div *ngFor="let order of orders" class="card">
+      <div class="emptyCard">
+        <img *ngIf="order.extend" src="assets/img/ubs-admin-orders/arrow_down.svg" alt="arrow down" (click)="changeCard(order.id)" />
+        <img *ngIf="!order.extend" src="assets/img/ubs-admin-orders/arrow_up.svg" alt="arrow up" (click)="changeCard(order.id)" />
+        <div>{{ order.order }}</div>
+        <div>{{ order.status }}</div>
+        <div class="orderSum">{{ order.sum }}</div>
+        <div class="btnsGroup">
```

Comment:
use snake case for class names
and fix this for all clasess

Reply:
Done